apps/web/src/components/form/Field.tsx

```diff
@@ -41,8 +43,10 @@ export const Field: React.FC<FieldProps> = props => {
         className={
           className ??
           classnames(
-            `w-full rounded-none bg-gray-100 block h-10
-            border-b-2 border-bcBlack pl-1 disabled:bg-bcDisabled`,
+            `w-full rounded-none block h-10
+            border-b-2 border-bcBlack pl-1 disabled:bg-bcDisabled ${
+              bgColour ? bgColour : 'bg-gray-100'
```

Comment:
design's grey is 'f5f5f5' but bg-gray-100 is 'f3f4f6'.
if you want to use tailwind, you can define a new one in `tailwind.config.js`